ngraph/python/src/pyngraph/node_factory.cpp

```diff
@@ -75,6 +75,7 @@ namespace
                 {"opset5", OpsetFunction(ngraph::get_opset5)},
                 {"opset6", OpsetFunction(ngraph::get_opset6)},
                 {"opset7", OpsetFunction(ngraph::get_opset7)},
+                {"opset8", OpsetFunction(ngraph::get_opset8)},
```

Comment:
Also this change should be also in separate PR.

Reply:
done